.github/workflows/build-and-quality-checks-v3.yml

```diff
@@ -43,3 +43,4 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           build_script: check:size
+          script: npm run check:size:json -- --scope '@rudderstack/analytics-js' --scope '@rudderstack/analytics-js-plugins'
```

Comment:
please change to npm run check:size , size checks are more specific and they need their builds in my branch

Reply:
Addressed.